### ISSUE TITLE
Navigating to website path that doesn't exist does not return 404

### ISSUE THREAD

**Issue by dchoi27** [nftstorage/nft.storage#1729]
Navigating to a part of the website that doesn't exist just takes you to the homepage, sometimes with some images broken. Let's get a 404 page up - can be a really simple design - I'll ask Protocol Labs design for one, but think we can start doing some digging on what is needed to get it up (something feels buggy about how the redirects are currently happening)

**Comment by dchoi27** [comment]
FYI - got a few designs back, I liked this one https://user-images.githubusercontent.com/97632209/161615773-825c022e-a9da-4845-8612-768dfe688225.png

Asking them why the background is different + seeing if they can get us over a Figma file

**Comment by dchoi27** [comment]
OK! They gave us back this asset for the 404 page: https://user-images.githubusercontent.com/97632209/161642459-f57b41b0-b12b-4023-ad43-0da3b501da59.gif

They said they went with the mint background because most of the pages had different colors, but I'm also OK with it being white (and keeping the nav bar orange). But in case helpful here's the mint version in GIF https://user-images.githubusercontent.com/97632209/161642459-f57b41b0-b12b-4023-ad43-0da3b501da59.gif

**Comment by JeffLowe** [comment]
Those 2 gifs look the same.  Was there a mint color version of the gif?

**Comment by JeffLowe** [comment]
I just noticed @dchoi27 the mint version asset gif have the nav and copy baked into the image.  
Can we please get just the blinking 404 portion?  We'll use the site header and actual text for "page not found". 

So we need this section:   

![Screen Shot 2022-04-08 at 9 07 12 AM](https://user-images.githubusercontent.com/2078567/162452563-1814018c-fd22-4e96-b619-d6ebc735a726.png)

**Comment by dchoi27** [comment]
Isn't that what's here? https://github.com/nftstorage/nft.storage/issues/1729#issuecomment-1088189939

**Comment by JeffLowe** [comment]
No.  We need the mint colored gif of only the 404 graphic (matching the screenshot image in my comment above).  The current mint version has a static image of the site navigation and the page copy "Page Not Found" baked into it.

**Comment by dchoi27** [comment]
like discussed here's a transparent version with no text https://user-images.githubusercontent.com/97632209/162500102-13bbeab4-dc3e-4193-b5ff-d286803faa37.gif

**Comment by johnathonroach** [comment]
@JeffLowe @dchoi27 I have an initial page with the graphic put together.

<img width="1665" alt="Screen Shot 2022-04-13 at 9 39 18 PM" src="https://user-images.githubusercontent.com/1884440/163303759-cd5a7223-2500-4f0a-bed4-098897ab14ea.png">

When I first pulled from the repo, there was a plain version of the 404 page already in place. Appears to have been added in #432 back in September. (https://github.com/nftstorage/nft.storage/pull/432/files#diff-bdc87f122600df1af57c9bbaa89a0b66ae51381f8312346645571e93c980cc59)

From there, I mostly applied styles and the image to the existing page. Things seem to be working but I am curious what types of paths I can test to make sure we have coverage. Jeff, you mentioned something about possibly testing pages from the docs section?

If nothing comes to mind, I can create a PR and we can testing things out.
